client/signup/steps/intent/intent-screen.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from '@automattic/components';
+import { Icon } from '@wordpress/icons';
+import { localize, LocalizeProps } from 'i18n-calypso';
+import React from 'react';
+// import intentImage from 'calypso/assets/images/intent-screen/intent.svg';
+import { build, write } from './icons';
+import type { IntentFlag } from './types';
+import './intent-screen.scss';
+
+interface Intent {
+	title: string;
+	description: string;
+	icon: React.ReactElement;
+	intent: IntentFlag;
+	actionText: string;
+}
+
+interface Props {
+	onSelect: ( intent: IntentFlag ) => void;
+	translate: LocalizeProps[ 'translate' ];
+}
+
+const useIntents = ( { translate } ): Intent[] => {
+	return [
+		{
+			title: translate( 'Write' ),
+			description: translate( 'Share your ideas with the world' ),
+			icon: write,
+			intent: 'write',
+			actionText: translate( 'Start writing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Start Writing' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/intent-screen)